src/vorta/views/ssh_dialog.py

```diff
@@ -68,15 +70,16 @@ def generate_key(self):
             self.sshproc.finished.connect(self.generate_key_result)
             self.sshproc.start('ssh-keygen', ['-t', format, '-b', length, '-f', output_path, '-N', ''])
 
-    def generate_key_result(self, exitCode, exitStatus):
-        if exitCode == 0:
+    def generate_key_result(self, exit_code):
```

Comment:
```suggestion
    @pyqtSlot(int)
    def generate_key_result(self, exit_code):
```

Not sure whether that is needed. But it might be since the finished signal can be emitted with two values.